Assets/Scripts/Game/UnderwaterFog.cs

```diff
@@ -18,8 +14,6 @@ public class UnderwaterFog
         public UnderwaterFog()
         {
             mainCamera = GameManager.Instance.MainCamera;
-            globalFog = mainCamera.GetComponent<GlobalFog>();
-            globalFog.enabled = true;
```

Comment:
I don't think fog is ever disabled. In that case i believe `RenderSettings.fog = true` should be the way to do it.

Reply:
That's fine. We'll need to evaluate any fog changes a bit differently now anyway.